x/pairing/client/cli/tx_stake_provider.go

```diff
@@ -68,6 +73,15 @@ func CmdStakeProvider() *cobra.Command {
 				return err
 			}
 
+			operatorFromFlag, err := cmd.Flags().GetString(types.FlagOperator)
+			if err != nil {
+				return err
+			}
+			operator, err := parseOperator(clientCtx, operatorFromFlag)
```

Comment:
you can use the new method you intoduced with the other PR

Reply:
done. see https://github.com/lavanet/lava/commit/8fa736e4fc672043c1744feec22aa8bc6ebec4ed